pkg/cmd/dev/roachprod_stress.go

```diff
@@ -23,6 +24,10 @@ const (
 	clusterFlag = "cluster"
 )
 
+var (
+	dependsOnGeos = false
```

Comment:
No mutable global variables please.

Reply:
Sure, doesn't need to be global anyways 🤷